WasatchNET/AndorEEPROMJSON.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace WasatchNET
+{
+    public class AndorEEPROMJSON
```

Comment:
This is an exact copy of RamanSpecCal.AndorEEPROMJSON.  As such, we should probably delete the file in RamanSpecCal, since that application can simply use this one instead (while the converse is not true).

Reply:
lets do that pre-merge

Reply:
I think we have to merge this first, so WPSC can see this new class. We can't delete the old class from WPSC until this new class is available.

Reply:
I'll go ahead and do it to remind myself

Reply:
jk it has 0 references in WPSC anyway, no worries at all